media/ckeditor/ckeditor/config.js

```diff
@@ -4,7 +4,20 @@
  */
 
 CKEDITOR.editorConfig = function( config ) {
-	// Define changes to default configuration here. For example:
-	// config.language = 'fr';
-	// config.uiColor = '#AADC6E';
+	config.toolbar = 'MyToolbar';
+ 
+    config.toolbar_MyToolbar =
+    [
+        { name: 'document', items : [ 'Source' ] },
+        { name: 'clipboard', items : [ 'Cut','Copy','Paste','PasteText','PasteFromWord','-','Undo','Redo' ] },
+        { name: 'editing', items : [ 'Find','-','Scayt' ] },
+        { name: 'insert', items : [ 'Image','Table','HorizontalRule','Smiley','SpecialChar','PageBreak'
+                 ,'Iframe' ] },
+                '/',
+        { name: 'styles', items : [ 'Styles','Format' ] },
+        { name: 'basicstyles', items : [ 'Bold','Italic','Underline','Strike','Subscript','Superscript','-','RemoveFormat' ] },
+        { name: 'paragraph', items : [ 'NumberedList','BulletedList','-','Outdent','Indent','-','Blockquote' ] },
+        { name: 'links', items : [ 'Link','Unlink','Anchor' ] },
+        { name: 'tools', items : [ 'Maximize','-','About' ] },
```

Comment:
Trailing comma. Do we generally still care about the browsers that this breaks on? It's pure reflex for me at this point...